blackboard.lua

```diff
@@ -0,0 +1,82 @@
+---@meta
+Blackboard = Blackboard or {}
+
+---In GAME section of the blackboard, add "Version=(int)3"
```

Comment:
I wonder if this varies between versions of the game?

Reply:
likely not. just checked my 210 exefs and it's also 3